### PR TITLE
New SKU support for MSN4600C

### PR DESCRIPTION
**Pull request by shlomibitton**
- What I did
Add SKU files for MSN4600C platform

- How I did it

- How to verify it

- Description for the changelog

### CODE REVIEW

File: device/mellanox/x86_64-mlnx_msn4600c-r0/ACS-MSN4600C/port_config.ini [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Ethernet280     280,281,282,283     etp36     35
+Ethernet288     288,289,290,291     etp37     36
+Ethernet296     296,297,298,299     etp38     37
+Ethernet304     304,305,306,307     etp39     38
+Ethernet312     312,313,314,315     etp40     39
+Ethernet320     320,321,322,323     etp41     40
+Ethernet328     328,329,330,331     etp42     41
+Ethernet336     336,337,338,339     etp43     42
+Ethernet344     344,345,346,347     etp44     43
+Ethernet352     352,353,354,355     etp45     45
```
Comment (keboliu) [inline device/mellanox/x86_64-mlnx_msn4600c-r0/ACS-MSN4600C/port_config.ini:46]: starting from this row, the index is wrong, should be 44 ~ 63

File: device/mellanox/x86_64-mlnx_msn4600c-r0/sensors.conf [hunk 2]
```diff
@@ -0,0 +1,156 @@
+################################################################################
+# Copyright (c) 2019 Mellanox Technologies
+#
+# Platform specific sensors config for SN4600
+################################################################################
+
+# Temperature sensors
```
Comment (keboliu) [inline device/mellanox/x86_64-mlnx_msn4600c-r0/sensors.conf:7]: need to ignor the fan2 and fan3 of PSU and please also check  the output in my email to see what needed to be changed.

File: device/mellanox/x86_64-mlnx_msn4600c-r0/ACS-MSN4600C/port_config.ini (hunk 1 above)
Comment (shlomibitton) [reply device/mellanox/x86_64-mlnx_msn4600c-r0/ACS-MSN4600C/port_config.ini]: fixed

File: device/mellanox/x86_64-mlnx_msn4600c-r0/sensors.conf (hunk 2 above)
Comment (shlomibitton) [reply device/mellanox/x86_64-mlnx_msn4600c-r0/sensors.conf:7]: fixed

**Comment by shlomibitton** [comment]
@keboliu 
please review

File: device/mellanox/x86_64-mlnx_msn4600c-r0/ACS-MSN4600C/sai_4600C.xml [hunk 3]
```diff
@@ -0,0 +1,467 @@
+<?xml version="1.0"?>
+<root>
+    <platform_info type="4601">
+
+        <!-- Device MAC address  -->
+        <device-mac-address>00:02:03:04:05:80</device-mac-address>
+
+        <!-- Number of ports in the following port list -->
+        <number-of-physical-ports>64</number-of-physical-ports>
```
Comment (nazariig) [inline device/mellanox/x86_64-mlnx_msn4600c-r0/ACS-MSN4600C/sai_4600C.xml:9]: @shlomibitton have you forgot to add issue?

Comment (shlomibitton) [reply device/mellanox/x86_64-mlnx_msn4600c-r0/ACS-MSN4600C/sai_4600C.xml:12]: added

**Comment by shlomibitton** [comment]
@nazariig @keboliu
please review

**Pull request closed without merging**